js/extensions/bindings/datatableBinding.js

```diff
@@ -298,7 +298,7 @@ define([
 			table.clear();
 
 			// Rebuild table from data source specified in binding
-			if (data.length > 0)
+			if (data && data.length > 0)
```

Comment:
This change is unrelated to the referenced issue. This fixes an error whereby the `data` object is not defined and so calling .length causes an error in the browser.